mne/utils/docs.py

```diff
@@ -3811,10 +3815,21 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
                       eog=250e-6      # unit: V (EOG channels)
                       )
 
-    .. note:: Since rejection is based on a signal **difference**
-              calculated for each channel separately, applying baseline
-              correction does not affect the rejection procedure, as the
-              difference will be preserved.
+    Custom rejection criteria can be also be used by passing a callable
+    to the dictionary.
+
+    Example::
+
+        reject = dict(eeg=lambda x: True if (np.max(x, axis=1) >
+                      1e-3).any() else False))
```

Comment:
Simplify (no need for True if x then False... that's the same as `bool(x)` which NumPy gives you for free anyway), make it an example someone might want to use, make it a single line to render better in Sphinx, and don't word as "as the dictionary":
```suggestion
    Custom rejection criteria can be also be used by passing a callable,
    e.g., to check for 99th percentile of absolute values of any channel
    across time being bigger than 1mV::

        reject = dict(eeg=lambda x: (np.percentile(np.abs(x), 99, axis=1) > 1e-3).any())
```
You might need to add a `# noqa: E501` at the end of the `"""` for this docstring